.github/workflows/ci-testing.yml

```diff
@@ -20,51 +20,72 @@ jobs:
         python-version: [3.6, 3.8]
         requires: ['minimal', 'latest']
         topic: [['devel']]
+        release: [ 'stable' ]
+        exclude:
+          - os: ubuntu-20.04
+            python-version: 3.8
+            requires: 'latest'
         include:
```

Comment:
I would set some base matrix with stable and add include as picking specific combinations